src/main/src/main/java/org/geoserver/catalog/impl/ResolvingProxy.java

```diff
@@ -22,7 +23,7 @@
  * @author Justin Deoliveira, The Open Planning Project
  *
  */
-public class ResolvingProxy extends ProxyBase {
+public class ResolvingProxy extends ProxyBase implements Serializable {
```

Comment:
Is this really part of the same (feedback panel) change?

Reply:
No, it's a fix for an issue I've found while developing this that was preventing me from moving on (it's related to dangling references in configuration). 
Will revert and someone else with time and interest will do it the "right" way.